### ISSUE TITLE
Problem opening a second session

### ISSUE THREAD

**Issue by kaspoz** [sienori/Tab-Session-Manager#1276]
## Short description
The extension no longer opens any saved sessions in a second window but only opens them in the current one

## Steps to reproduce

1. Start browser with clean profile
2. Install Tab Session Manager
3. 
4. 

## Platform information
* Platform (OS): Windows 10
* Version of browser: 121.0 (64bit)
* Version of Tab Session Manager: 6.12.2

**Comment by pgeddes** [comment]
Yes I also have this problem my version is 6.12.2 which last updated 15 September 2023.
The "open in new window" , which I have as default, does not work, it also does not work in the right click context menu.
It used to work before, so it might have stopped working after the update, I am not sure.
